pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -601,3 +601,34 @@ def _get_sorter(theme_code):
         sorter = Renderer._load_sorter(sorter_config['module'], sorter_config['class_name'])
         params = sorter_config.get('params', {})
         return sorter, params
+
+    def _get_sorted_legend(self, legend_list):
+        """
+        Takes an unsorted list of legend and returns it sorts, depending on the geometry type
+        and on the value of the geomety tye if this is available
+
+        Args:
+            legend_list: list of legends
+
+        Returns:
+             sorted list of legends
+        """
+        return sorted(legend_list, key=self._sort_legend_elem)
+
+    @staticmethod
+    def _sort_legend_elem(elem):
+        """
+        Returns a tuple of the geometry type of the element and the value of the geometry
+        type if it is available. Else it returns the geometry type and 0 as value
+
+        Args:
+            elem: one legend entry
+
+        Returns:
+            geom_type, int value
+        """
+        geom_type = elem['Geom_Type']
+        if geom_type == 'NrOfPoints':
+            return (geom_type, 0)
+        else:
+            return geom_type, elem[geom_type]
```

Comment:
Descending order might be better, because the larger values are more relevant?

`return geom_type, -elem[geom_type]`